packages/eslint-config-addin/src/main.ts

```diff
@@ -0,0 +1,8 @@
+export = {
+    rules: {
+        "no-eval": "error",
```

Comment:
Are these the rules based on the security recommendations? It would be good to have the readme explain the rationale for our choices, although that's not needed to complete this check-in. 

Another question is whether our config should also extend a standard eslint config.

Reply:
Yes these are the rules given to me.  With this config I was just focusing on the security issues as the standard configs are way more opinotated and are not related to security.

Reply:
![image](https://user-images.githubusercontent.com/741153/62567172-80828a80-b83f-11e9-9d97-aab3889468bd.png)

Let's have this extend from "eslint:recommended".  We can then see how it looks and tweak from there.

Reply:
We might need to also need to have more than one config, for differences with React or Angular.

Reply:
added the extends from eslint:recommended.  Will look into created more configs based on the framework used.